packages/bot-web-ui/src/stores/dashboard-store.ts

```diff
@@ -1,43 +1,93 @@
-import { observable, action } from 'mobx';
+import { observable, action, reaction } from 'mobx';
 import RootStore from './root-store';
 
+const clearInjectionDiv = () => {
+    const el_ref = document.getElementById('load-strategy__blockly-container');
+    if (el_ref?.getElementsByClassName('injectionDiv').length > 1) {
+        el_ref.removeChild(el_ref.getElementsByClassName('injectionDiv')[0]);
+    }
```

Comment:
```suggestion
    const el_ref = document.getElementById('load-strategy__blockly-container');
    if (el_ref?.getElementsByClassName('injectionDiv').length > 1) {
        el_ref.removeChild(el_ref.getElementsByClassName('injectionDiv')[0]);
    }
```
👀

Reply:
could not find a method to clear from blockly to remove injection div so used plain JS